docs/README.md

```diff
@@ -124,17 +132,40 @@ productSearch(
     If you want faster searches and do not care about most up to date prices and promotions, use skip value.
     """
     simulationBehavior: SimulationBehavior = default
+    """
+    If true, all the products will be solved by VTEX API.
+    """
+    productOriginVtex: Boolean = false
+    """
+    Indicates how the search-engine will deal with the fullText.
+    """
+    operator: Operator
+    """
+    Indicates how the search engine will correct misspeled words.
+    """
+    fuzzy: String
+    """
+    It is similar to fuzzy and operator but is used for scenarios where fuzzy and operator are not enough.
+    """
+    searchState: String
   ): ProductSearch
   ```
 
   ```
   type ProductSearch {
-  products: [Product]
-  recordsFiltered: Int
-  titleTag: String
-  metaTagDescription: String
-  breadcrumb: [SearchBreadcrumb]
-}
+    products: [Product]
+    recordsFiltered: Int
+    titleTag: String
+    metaTagDescription: String
+    breadcrumb: [SearchBreadcrumb]
+    canonical: String
+    suggestion: SearchSuggestions
+    correction: SearchCorrection
```

Comment:
What's the difference between suggestion and correction?

Reply:
I've updated the [`search-protocl`](https://github.com/vtex/search-protocol/blob/master/docs/queries/productSearch.md#response) to make it clear.

Suggestion refers to suggestions related to the fullText term. For example, if you search for `shirt` we could suggest you `blue shirt` or `t shirt`. Correction is the "Did you mean" feature, for example, to correct `camizeta` to `camiseta`.

Reply:
Thanks!